CHANGELOG.rst

```diff
@@ -47,6 +47,13 @@
   `#1629`_. Thanks `@obestwalter`_ and `@davehunt`_ for the complete PR
   (`#1633`_)
 
+* New cli flag ``--override-ini`` or ``-o`` that overrides ini values from the
+command line so that one can do e.g. "-o xfail_strict=True". This is useful for
+specifying options for which there is no separate command-line flag, a complete
+ini-options can be showed by py.test --help
```

Comment:
i'd make a "." after flag. and then say "The complete list of ini-options can be shown by ``py.test --help``". Mind the escape of the command which is important for rendering the docs.